aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -702,72 +657,61 @@ module std::features {
         false
     }
 
-    /// Whether function values are enabled.
-    /// Lifetime: transient
-    ///
-    /// We do not expect use from Move, so for now only for documentation purposes here
-    const ENABLE_FUNCTION_VALUES: u64 = 89;
```

Comment:
same here. could you please do a full check to see if anything is deleted by accident?

Reply:
Yes, it should be correct now only adding `STAKE_REWARD_USING_TREASURY: u64 = 224;`